Vagrantfile

```diff
@@ -92,6 +98,26 @@ settings_merger = proc {
     end
 }
 
+def check_and_select_conf_options(selected_dict,target_key,option_array,error)
```

Comment:
Can you add a comment that explains what this function does?

Reply:
done